keyboards/splitty/config.h

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 Nicolas Schodet
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x1d50
+#define PRODUCT_ID 0x6052
+#define DEVICE_VER 0x0001
+#define MANUFACTURER Nicolas Schodet
+#define PRODUCT Splitty
+
+/* key matrix size */
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 7
+
+/* Keyboard Matrix Assignments */
+#define MATRIX_ROW_PINS \
+    { F0, F1, D4, D5, D6 }
+#define MATRIX_COL_PINS \
+    { B0, B1, B2, B3, B4, B5, B6 }
+#define DIODE_DIRECTION COL2ROW
+
+/* Split Keyboard specific options. */
+#define USE_I2C
+#define SOFT_SERIAL_PIN D0
+#define EE_HANDS
+#define SPLIT_USB_DETECT
```

Comment:
Does the pcb lack VBUS detection?  If not, then this isn't needed.  Specifically, this is only needed if it lacks proper VBUS detection

Reply:
Yes, VBUS is connected to VCC.

Reply:
If that's the case, then this isn't needed.  The `SPLIT_USB_DETECT` define is only needed if VBUS isn't correctly set up, and can't be used.

Reply:
The part which is not connected to USB receives its power from the “TRRS”, and this is connected to VBUS, so it believes USB is connected. As far as I know, other boards use an extra voltage regulator so that VBUS and VCC are not directly connected.

Reply:
If that's the case, then you shouldn't need this setting, as this is only for boards that lack VBUS detection (such as a majority of supported ARM boards, or the v3.0 of the Elite C).

Reply:
My boards lacks VBUS detection because VCC is directly connected to VBUS, it can not make the difference between connected by USB and powered by the connection between the boards.

Without `SPLIT_USB_DETECT`, both parts think they are connected to USB and I have only one half working.